jans-cedarling/cedarling/src/lib.rs

```diff
@@ -27,9 +27,20 @@ mod tests;
 use std::rc::Rc;
 
 use authz::Authz;
-use log::init_logger;
+use init::policy_store::{load_policy_store, ErrorLoadPolicyStore};
 pub use log::LogStorage;
+use log::{init_logger, LogWriter};
 pub use models::config::*;
+use models::log_entry::{LogEntry, LogType};
+use uuid7::uuid4;
+
+/// Errors that can occur during initialization Cedarling.
+#[derive(Debug, thiserror::Error)]
+pub enum ErrorInitCedarling {
```

Comment:
`InitCedarlingError`, the name usually comes first, suffixed by `Error`

Reply:
done